com.github.arshubham.gitignore.json

```diff
@@ -0,0 +1,41 @@
+{
+    "app-id": "com.github.arshubham.gitignore",
+    "runtime": "org.gnome.Platform",
+    "runtime-version" : "3.38",
+    "base" : "io.elementary.BaseApp",
+    "base-version" : "juno-20.08",
+    "sdk": "org.gnome.Sdk",
+    "command": "com.github.arshubham.gitignore",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--share=network",
+        "--filesystem=home"
```

Comment:
What does it need home access for?

Reply:
That's for saving the gitignore files.

Reply:
A NativeFileChooser should work just fine without the explicit home permission.

Reply:
I tried doing that, but it doesn't allow the saving of files into the system